rear_rider_device/cam_unit_test.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import rear_rider_sensors.camera as camera
+import os
+
+class CameraUnitTest(unittest.TestCase):
+
+    cam : camera.RRCamera = None
+
+    def test_picture(self):
+        pic_name = "PhotoUnitTest"
+        self.cam.takePhoto(pic_name)
+        file_exists = os.path.exists(os.path.dirname(__file__) + "/media_storage/" + pic_name + '.jpg')
+        self.assertTrue(file_exists)
+
+    def test_recording(self):
+        vid_name = "VideoUnitTest"
+        self.cam.startRec(vid_name)
```

Comment:
It would be nice to set how long the video records for here so that we do not have to spend 15 seconds every time this test runs. One second should be fine enough since the only test here is checking if the video file is actually being created and nothing more.

Reply:
curious cause i haven't run this yet; why does this take 15 seconds?

alternative you could stop recording immediately after starting - less than 1 second!

Reply:
> curious cause i haven't run this yet; why does this take 15 seconds?

That is the default recording length defined in one of the classes that was implemented.